kubernetes/template/DeploymentTemplates/script.sh

```diff
@@ -208,6 +222,68 @@ apt_get_install()
     wait_for_apt_locks
 }
 
+generate_api_model()
+{
+    touch $DEFINITION_TEMPLATE_NAME
+    cat > $DEFINITION_TEMPLATE_NAME <<EOL
+    {
+    "apiVersion": "vlabs",
+    "location": "",
+    "properties": {
+        "orchestratorProfile": {
+            "orchestratorType": "Kubernetes",
+            "orchestratorRelease": "",
```

Comment:
FYI, we may have to specify `orchestratorVersion` as well. AKSe v0.48.0 supports 1.15.11 and our latest is 1.15.10